docs/first-steps/diving-deeper/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Exploring Snowplow beyond the first steps"
+sidebar_position: 7
+sidebar_label: "Diving deeper"
+description: "What to look at once you are familiar with Snowplow basics"
+---
+
+Here are a few suggestions for your next steps:
+* Browse the [recipes](/docs/recipes/index.md) — short guides for common use cases
```

Comment:
FYI - most of the recipes are written specifically for Try Snowplow so use only PostgreSQL and reference Try directly. Dont think it changes anything for this iteration but something we need to keep in mind to not frustrate non-Try users

Reply:
At the time when Cloud was released, we made a copy of these specifically for Cloud. At some point I merged them together, so that the same recipes work for both Try and Cloud. The only two that are Try-specific are the querying one and the failed events one (both of which I did mark as Try-specific)